README.md

```diff
@@ -59,6 +59,10 @@ Build instructions:
 	cd build
 	make
 
+(Note: do_cmake.sh now defaults to creating a debug build of ceph that can
+be up to 5x slower with some workloads. Please pass 
+"-DCMAKE_BUILD_TYPE=Release" to do_cmake.sh to create a non-debug release.)
```

Comment:
i'd suggest s/Release/RelWithDebInfo/ in this changeset to ensure that user will have debug symbol generated.

Reply:
+1

Reply:
I'd also modify do_cmake.sh itself to print this out in a loud/noticable way at the top or bottom
```
****
  WARNING: debug build selected -- performance will be bad but gdb will be happy
****
```
or something